src/script/update-script.rb

```diff
@@ -89,12 +89,32 @@
 # https://github.com/wemake-services/kira-dependencies/pull/210
 excluded_requirements = ENV['DEPENDABOT_EXCLUDE_REQUIREMENTS_TO_UNLOCK']&.split(" ")&.map(&:to_sym) || []
 
+#################################
+# Setup the protocol to be used #
+#################################
+protocol = ENV["AZURE_PROTOCOL"] || "https"
+puts "Using '#{protocol}' as protocol"
+
 #################################
 # Setup the hostname to be used #
 #################################
 azure_hostname = ENV["AZURE_HOSTNAME"] || "dev.azure.com"
 puts "Using '#{azure_hostname}' as hostname"
 
+
+#################################
+# Setup the port to be used #
+#################################
+port = ENV["AZURE_PORT"] || "443"
```

Comment:
We may want to use `80` as the default port, if `protocol == "http"`. Otherwise, some `"http://your.tfs:443/collection"` URL might be constructed later.

Reply:
makes sense, updated it